detector/terraform_module_pinned_source.go

```diff
@@ -10,18 +10,16 @@ import (
 
 type TerraformModulePinnedSourceDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	DeepCheck  bool
 }
 
 func (d *Detector) CreateTerraformModulePinnedSourceDetector() *TerraformModulePinnedSourceDetector {
-	return &TerraformModulePinnedSourceDetector{
-		Detector:   d,
-		IssueType:  issue.WARNING,
-		TargetType: "module",
-		DeepCheck:  false,
-	}
+	nd := &TerraformModulePinnedSourceDetector{Detector: d}
+	nd.Name = "terraform_module_pinned_source"
+	nd.IssueType = issue.WARNING
+	nd.TargetType = "module"
```

Comment:
__[gometalinter]__
* [goconst] 3 other occurrence(s) of "module" found in: test_helper.go:81:7 detector.go:169:7 detector.go:238:7

<!-- Comment by [SideCI](https://sideci.com) -->